pkg/expapi/v1/defaults.go

```diff
@@ -16,5 +16,13 @@ limitations under the License.
 
 package v1
 
+import "k8s.io/kubernetes/pkg/api"
+
 func addDefaultingFuncs() {
+	api.Scheme.AddDefaultingFuncs(
+		func(obj *APIVersion) {
```

Comment:
This defaulting function is also added in api/v1/defaults.go. I think the duplication is unnecessary.